single_kernel_mongo/events/lifecycle.py

```diff
@@ -175,6 +183,8 @@ def on_config_changed(self, event: ConfigChangedEvent):
                 scope="unit",
                 component=self.dependent.name,
             )
+        except (NonDeferrableFailedHookChecksError, WorkloadServiceError):
```

Comment:
Same, better be verbose in logging than leaving an empty return.
If not, the user won't know why the event did not end up in what they where expecting.